src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -1,699 +1,573 @@
+                      <li data-questiontype="CONTRIB"><a href="javascript:;">Team contribution question</a></li>
+                      <li data-questiontype="RUBRIC"><a href="javascript:;">Rubric question</a></li>
+                      <li data-questiontype="RANK_OPTIONS"><a href="javascript:;">Rank (options) question</a></li>
+                      <li data-questiontype="RANK_RECIPIENTS"><a href="javascript:;">Rank (recipients) question</a></li>
+                    </ul>
+                    <a target="_blank" rel="noopener noreferrer">
+                      <i class="glyphicon glyphicon-info-sign"></i>
+                    </a>
+                    <a id="button_copy" class="btn btn-primary margin-bottom-7px" data-actionlink="/page/instructorFeedbackQuestionCopyPage?user=test%40example.com" data-fsname="hgc" data-courseid="teammates.instructor.uni-demo" data-target="#copyModal" data-toggle="modal">
+                      Copy Question
+                    </a>
+                    <a id="button_done_editing" class="btn btn-primary margin-bottom-7px">
+                      Done Editing
+                    </a>
+                  </div>
+                </div>
               </div>
             </div>
           </div>
-        </div>
-      </div>
-
-      <div class="row">
-        <div class="col-sm-6">
-          <p>There are several question types to choose from.
-            <br>See
-            <a href="#fbQuestionTypes">here</a> for more information on each question type.
+          <p>
+            TEAMMATES gives you fine-grained control of each question. In addition to providing a range of different
+            <a href="#fbQuestionTypes">question types</a>, you can also customize your desired:
           </p>
           <ul>
             <li>
-              <a href="#fbEssay">Essay question</a>
-            </li>
-            <li>
-              <a href="#fbMcq">Multiple-choice (single answer) question</a>
-            </li>
-            <li>
-              <a href="#fbMsq">Multiple-choice (multiple answers) question</a>
+              <b>Feedback Path</b>: define who is giving feedback, and who the feedback is about.
+              Select a common feedback path from the dropdown menu, or choose "Other predefined combinations..."
+              to define the Feedback Giver and Recipient separately.
+              If you choose a ‘team’ as the giver, any member can submit the response on behalf of the team.
             </li>
             <li>
-              <a href="#fbNumscale">Numerical-scale question</a>
-            </li>
-            <li>
-              <a href="#fbConstSumOptions">Distribute points (among options) question</a>
-            </li>
-            <li>
-              <a href="#fbConstSumRecipients">Distribute points (among recipients) question</a>
-            </li>
-            <li>
-              <a href="#fbContrib">Team contribution question</a>
-            </li>
-
```

Comment:
I know this was in the previous UI. This info-sign looks quite confusing to me- I couldn't tell it was part of the UI for the question, or a genuine reminder to the instructors to save the changes in the instructor help page. I think the purpose here could be achieved without using the green colour alert box.